crud.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// Inserting the record
+func (apiCfg apiConifg) insertRecord(record any) (interface{}, error) {
+	// Maybe extract the dbName and the collectionName from .env file?
+	collection := apiCfg.dbClient.Database("munch").Collection("Testing")
+	// Do i wanna return the ObjectId?
+	result, err := collection.InsertOne(context.TODO(), record)
+	if err != nil {
+		return nil, err
+	}
+	log.Printf("Inserted record with _id: %v", result.InsertedID)
+	return result.InsertedID, nil
+}
+
+// Reading a record, given an Object ID
+func (apiCfg apiConifg) readRecord(objectId any) *mongo.SingleResult {
+	collection := apiCfg.dbClient.Database("munch").Collection("Testing")
```

Comment:
Repeating

as mention by @raamsri

we have to wrap this code and create a function which does these.